template_init_data.json

```diff
@@ -5,23 +5,26 @@
             "data": {
                 "id": "DY1cscbWCk3F4_ZUakA3B",
                 "title": "The Cyberiad",
-                "author": "Stanislaw Lem"
+                "author": "Stanislaw Lem",
+                "numPages": 295
```

Comment:
Are these the actual number of pages for each book?

Reply:
Yes.
I check it online.